### PR TITLE
Add support for BINARY field in MySQL

### PR DESCRIPTION
**Pull request by foxx**
See #801

### CODE REVIEW

**Comment by coleifer** [comment]
Needs tests in order to merge.

**Comment by coleifer** [comment]
You can add tests to the `test_fields.TestFieldTypes` or you can just create a one-off test-case. Either is fine.

**Comment by foxx** [comment]
Done. Apologies for the initial lack of tests, I got lazy!

**Comment by coleifer** [comment]
Those tests don't actually test the binary field.

**Comment by foxx** [comment]
Ah crap, it looks like my cherry pick apply properly due to lack of concentration, will get this fixed in the morning, sorry

**Comment by foxx** [comment]
Done, I'm pretty sure this is fine but any problems let me know

File: playhouse/tests/test_fields.py [hunk 1]
```diff
@@ -277,41 +277,49 @@ class CustomFormatsModel(Model):
...
     @skip_test_unless(lambda: isinstance(test_db, MySQLDatabase))
-    def test_blob_field_mysql(self):
-        data = bytes(bytearray(range(256)))
-        blob = BlobModel.create(data=Param(data))
-        res = BlobModel.get(BlobModel.id == blob.id)
-        self.assertEqual(blob.data, data)
+    def test_blob_and_binary_field_mysql(self):
+        def inner(cls):
+            data = bytes(bytearray(range(256)))
+            blob = cls.create(data=Param(data))
```
Comment (coleifer) [inline playhouse/tests/test_fields.py]: Why is this necessary?

**Comment by coleifer** [comment]
Thank you!
